observations/obs_converters/AIRS/amsua_netCDF_support_mod.f90

```diff
@@ -131,7 +138,8 @@ subroutine initialize_amsua_netcdf(filename)
 if ( AMSUA_BT_WARMPRTA2   /= WARMPRTA2   ) mismatch = .true.
 
 if (mismatch) then
-   call error_handler(E_ERR,'initialize_amsua_netcdf','assumptions wrong',source)
+   call error_handler(E_ERR,'initialize_amsua_netcdf','assumptions wrong',source,&
+              text2='dimensions in file do not match expected dimensions')
```

Comment:
FYI only: this particular check makes it impossible to use a netCDF file with a single pixel as test input